src/codeu/chat/client/ClientUser.java

```diff
@@ -59,23 +59,23 @@ static public boolean isValidName(String userName) {
     return clean;
   }
 */
-
+    
 // Validate the username string
 public boolean isValidName(String userName) {
+	boolean clean = true; 
```

Comment:
Another small nit in terms of name -- its not clear to me what the variable represents from reading this variable name. Maybe something like isUniqueUser or isNewUser?